src/ispc.h

```diff
@@ -327,7 +346,9 @@ class Target {
 
     bool hasFp64Support() const { return m_hasFp64Support; }
 
-    bool warnFtoU32IsExpensive() const { return m_warnFtoU32IsExpensive; }
+    void setWarning(PerfWarningType warningType) { m_warnings |= static_cast<unsigned int>(warningType); }
+
+    bool shouldWarn(PerfWarningType warningType) { return (m_warnings & static_cast<unsigned int>(warningType)) != 0; }
```

Comment:
I am slightly worried about that because enum is actually signed int.

Reply:
That should be an issue if the underlying type is picked smaller than `unsigned int`, so the conversion would involve sign extend. Default choice for the compiler in this case is `int`, but it's implementation defined. So you are right that it's a gray zone. I've pushed commit that fixes that.

Reply:
Great!